packages/ProgressAccordion/stories/examples/Showcase.js

```diff
@@ -7,7 +7,7 @@ import generatedItems from "../fixtures/generateItems";
 import ProgressAccordion from "../../src";
 
 const propKnobs = () => ({
-  activeIndex: select("activeIndex", [0, 1, 2, 3, 4], 2),
+  activeIndex: select("activeIndex", [0, 1, 2, 3, 4], 3),
```

Comment:
🤨why?

Reply:
😄 i was testing if the knobs were even working in progress accordion they are after the other change on line 52 
`export default () => <ExampleStory {...propKnobs()} />;`